.travis.yml

```diff
@@ -37,6 +37,8 @@ install:
   - pip install sphinx numpydoc
   # Install PEP 8 linters (flake8 is like pep8 but also reports unused imports)
   - pip install flake8
+  # Install matplotlib for benchmarks
+  - pip install matplotlib
```

Comment:
Installing matploblib with pip is generally a bad idea. If you want to install it, install it with apt-get.